File: Java/1080.java
// Problema: 1080 - Maior e Posição | Resposta: Accepted
// Linguagem: Java 19 [+2s]         | Tempo: 0.115s

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num, maior = -1, pos = 0;
        for(int i = 0; i < 100; i++) {
        	num = Integer.parseInt(br.readLine());
        	if(num > maior) {
        		maior = num;
        		pos = i;
        	}
        }
        
        pos++;
        System.out.println(maior);
        System.out.println(pos);
    }
}

File: C/1080.c
// Problema: 1080 - Maior e Posição | Resposta: Accepted
// Linguagem: C99 [+0s]             | Tempo: 0.000s

#include <stdio.h>

int main() {
    int num, maior = -1, pos;

    for(int i = 0; i < 100; i++){
        scanf("%d", &num);
        if(num > maior){
            maior = num;
            pos = i + 1;
        }
    }

    printf("%d\n%d\n", maior, pos);
    return 0;
}